osgar/tools/lidarview.py

```diff
@@ -480,6 +484,7 @@ def lidarview(gen, caption_filename, callback=False, out_video=None, jump=None):
                 if event.key == K_9:
                     frames_step = 90
                     sleep_time = 10
+                sleep_time = 0
```

Comment:
I guess this is forgotten mistake? (otherwise previous like for `K_9` does not have much sense ...

Reply:
No, that is the intention. That is the thing I am asking about. I prefer not to have any delays. K_9 primarily controls how many frames will be skipped. As I state at the beginning of the PR description

> I'd like to discuss removing the time between rendering the frames.

The PR is a draft - so it is minimal code change to demonstrate the behavior I would prefer. Once we agree on the behavior, I will adjust the code so it is pretty and ready for merge.